services/bots/src/github-webhook/handlers/issue_comment_commands/commands.ts

```diff
@@ -0,0 +1,85 @@
+import { IssueCommentCreatedEvent } from '@octokit/webhooks-types';
+import { WebhookContext } from '../../github-webhook.model';
+import { IntegrationManifest } from '../../utils/integration';
+import { IssueCommentCommand, IssueCommentCommandContext } from './const';
+
+export const ISSUE_COMMENT_COMMANDS: { [command: string]: IssueCommentCommand } = {
+  close: {
+    description: 'Closes the issue.',
+    invokerType: 'code_owner',
+    handler: async (
+      context: WebhookContext<IssueCommentCreatedEvent>,
+      command: IssueCommentCommandContext,
+    ) => {
+      if (invokerIsCodeOwner(command)) {
+        throw new Error('Only code owners can close issues.');
+      }
+
+      await context.github.issues.update(
+        context.issue({
+          state: 'closed',
+        }),
+      );
+    },
+  },
+  rename: {
+    description: 'Change the title of the issue.',
+    exampleAdditional: 'Awesome new title',
+    invokerType: 'code_owner',
+    requireAdditional: true,
+    handler: async (
+      context: WebhookContext<IssueCommentCreatedEvent>,
+      command: IssueCommentCommandContext,
+    ) => {
+      if (invokerIsCodeOwner(command)) {
```

Comment:
Same
```suggestion
      if (!invokerIsCodeOwner(command)) {
```

Reply:
one more found while writing the tests for it <https://github.com/home-assistant/service-hub/pull/123/commits/0c2be776b1b43a49b35544a9348ca22b83cfc94c#diff-a75c2dcee499b4315a0bd7b2409e221cfadc4c0e5fdd09f370cf2b62550313fbR51>